cmd/limactl/stop.go

```diff
@@ -105,12 +105,12 @@ func waitForHostAgentTermination(ctx context.Context, inst *store.Instance, begi
 
 func stopInstanceForcibly(inst *store.Instance) {
 	if inst.QemuPID > 0 {
-		logrus.Infof("Sending SIGKILL to the QEMU process %d", inst.QemuPID)
+		logrus.Infof("Sending SIGKILL to the %s driver process %d", inst.VMType, inst.QemuPID)
```

Comment:
Should `QemuPID` be renamed to `DriverPID`?

Also `filenames.QemuPID` should then be renamed as well, and use a different filename. It is confusing to have a `qemu.pid` file, but no qemu process.

Reply:
QEMU driver should continue using `qemu.pid`, somebody might be potentially depending on it

Reply:
Exactly, i wanted to change this to driver.pid. But later realised that this might have impact on other current running instances as well (Stop may not work well, unless we support both files). So thought of doing it in a follow-up.

There are 2 ways i could see to solve this,
- Maybe introduce a way to get driver specific pid file (qemu.pid/vz.pid). The hostagent can use this driver specific method in places like stop, identify status. 
- deprecate qemu.pid and for qemu driver start creating both driver.pid and qemu.pid. We can clean qemu.pid once we decide when to remove (Maybe 1.0)

Am happy to do the required changes as part of this PR as well.

Reply:
> driver specific pid file (qemu.pid/vz.pid)

SGTM

Reply:
Good point! Followup PR is fine; this PR is big enough already! 😄

> Maybe introduce a way to get driver specific pid file (qemu.pid/vz.pid). The hostagent can use this driver specific method in places like stop, identify status.

I think I would prefer this way to "solve" the issue. Maybe have a method in the driver package that provides the pid filename?

Reply:
Yes exactly, driver can give out their own pid name. This will be used through out hostagent code-base.

I will take this up in a follow-up PR 👍